custom_components/easee/controller.py

```diff
@@ -214,6 +225,43 @@ async def schedules_async_refresh(self):
 
         _LOGGER.debug("Schedule: %s %s", self.schedule, self.weekly_schedule)
 
+    def schedules_interpret(self, data):
+        self.schedule_polled = True
+
+        startEpoch = data.get("StartSchedule", 0)
```

Comment:
```suggestion
        start_epoch = data.get("StartSchedule", 0)
```
Use snake_case for variable names - HA code style